src/core/methods.ts

```diff
@@ -14,6 +14,7 @@ export function selectPage(this: uibPagination, page: number, evt) {
             evt.target.blur();
         }
         this.currentPage = page;
+        if (this.changeOnClick) this.$emit('changed')
```

Comment:
Please make this changes:
+ Use brackets, semicolons and double quoutes.
``` js
if (this.changeOnClick) { this.$emit("change"); }
```
+ Use event `change` instead of `changed`. ¿Or did you mean to create another event?

Reply:
`change`. My bad